modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -146,6 +153,20 @@ const EditorUpload = function (editor: Editor): EditorUpload {
           };
         });
 
+        if (imagesToRemove.length > 0) {
+          if (Rtc.isRtc(editor)) {
+            // To be replaced by RTC API to mirror DOM changes when such is implemented.
+            console.error('Removing images on failed uploads is currently unsupported for RTC'); // eslint-disable-line no-console
+          } else {
+            editor.undoManager.transact(() => {
+              Arr.each(imagesToRemove, (element: HTMLImageElement) => {
```

Comment:
You don't need the type here, it's already inferred from the type on `imagesToRemove`, so can you please remove it?

Reply:
If you insist.